cpp/BUILD.bazel

```diff
@@ -4,14 +4,14 @@ load("@rules_pkg//:mappings.bzl", "pkg_filegroup")
 
 alias(
     name = "dbscheme",
-    actual = "//cpp/ql/lib:dbscheme",
+    actual = "@ql//cpp/ql/lib:dbscheme",
```

Comment:
Why do we need this? Is this needed for the integration into the main repo?

Reply:
Yeah I think there was an issue not doing this in semmle-code, but I'm not so sure any more. I'll try to revert and report back

Reply:
nope, could not find the reason, removed all `@ql` here